Week1/homework/js-excercises/e10-compare arrays.js

```diff
@@ -1,12 +1,27 @@
+// 'use strict'
+
+// const myArray1=['HYF', 25, '#bestclassever', {instructor: 'Wilgert'}]
+// const myArray2=['study', 'read', 'hangout', 'eat', 'drink', 'live', 101]
+
+// console.log(myArray1.length)
+// console.log(myArray2.length)
+
+// if (myArray1.length==myArray2.length) {
+//     console.log('Same size!')
+// } else{
+//     console.log('Two different sizes')
+// }
+
+
 'use strict'
 
-const myArray1=['HYF', 25, '#bestclassever', {instructor: 'Wilgert'}]
-const myArray2=['study', 'read', 'hangout', 'eat', 'drink', 'live', 101]
+const JS_class25=['HYF', 25, '#bestclassever', {instructor: 'Wilgert'}]
+const Hobbies=['study', 'read', 'hangout', 'eat', 'drink', 'live', 101]
```

Comment:
Remember to use `camelCase` here. So I would change `JS_class25` into `jsClass25`. And same for `Hobbies`